src/lib/components/about/officer-profile-list.svelte

```diff
@@ -62,9 +62,10 @@
   }
 
```

Comment:
We could make sure the parent has a desired width:

```suggestion
.container {
  width: 100%;
  max-width: 1200px;
}
```

Reply:
done!